lib/__tests__/standalone-fix.test.js

```diff
@@ -0,0 +1,256 @@
+'use strict';
+
+const _ = require('lodash');
+const del = require('del');
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+const stripIndent = require('common-tags').stripIndent;
+const { promisify } = require('util');
+
+const replaceBackslashes = require('../testUtils/replaceBackslashes');
+const standalone = require('../standalone');
+
+const copyFile = promisify(fs.copyFile);
+const readFile = promisify(fs.readFile);
```

Comment:
That's odd. I originally wrote it using the promise API but refactored it after the `node/no-unsupported-features` ESLint plugin warned against using it. I'll take another look because it [appears to be a stable API in node@10](https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_promises_api).

Reply:
https://github.com/mysticatea/eslint-plugin-node/issues/174

Reply:
I'll use the API and disable the ESLint warning for now.